web/tests/composebox_typeahead.test.js

```diff
@@ -30,6 +32,17 @@ const message_user_ids = mock_esm("../src/message_user_ids", {
 });
 const stream_topic_history_util = mock_esm("../src/stream_topic_history_util");
 
+mock_esm("../src/markdown_config", {
+    get_helpers() {
+        return {
+            stream_topic_hash: hash_util.by_stream_topic_url,
+            get_stream_by_name() {
+                return "Rome";
+            },
```

Comment:
This mock seems unnecessary; can't we just use add the stream to the `stream_data` database and avoid the hackery?

Reply:
I was unnecessarily trying to mimic the code path taken by the frontend markdown processor. 
I have simplified things by using the stream data already present and being used by other tests.

Reply:
Sure, I was unnecessarily trying to mimic the code path followed by the frontend markdown processor (which uses `markdown_config.ts`).

I have removed these and used the streams already present for the tests. This simplified the code a lot.